### ISSUE TITLE
cm-rgb-cli error when I try to change anything

### ISSUE THREAD

**Issue by diehard67** [gfduszynski/cm-rgb#6]
after updating to version 0.0.8 when I tried to play around a bit I got this error and no changes actually happen to my cooler.
0.0.5 and 0.0.7 worked fine

I am running ubuntu 19.10
I upgrade using pip
sudo pip install --upgrade cm-rgb

```
sudo cm-rgb-cli logo --mode=static --color=#ffffff --speed=3 --brightness=5 fan --mode=breathe --color=#ff0000  --speed=3 --brightness=5 ring --mode=swirl --color=#ffffff  --speed=1 --brightness=5 save
Traceback (most recent call last):
  File "/usr/local/bin/cm-rgb-cli", line 160, in <module>
    cli()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1164, in invoke
    return _process_result(rv)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1102, in _process_result
    **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/bin/cm-rgb-cli", line 26, in process_pipeline
    elif ctx.obj['version']:
KeyError: 'version'
```
other than this hiccup, I love that I can control my coolers RGB over USB in Linux, I don't even have windows on this computer.

**Comment by gfduszynski** [comment]
Thank you very much for reporting.
I've fixed this in 0.0.9.

I hope this cli will serve you well :)

**Comment by diehard67** [comment]
wow, that was fast, thanx
